burn-dataset/src/source/huggingface/downloader.rs

```diff
@@ -10,6 +10,10 @@ use thiserror::Error;
 
 const PYTHON: &str = "python3";
```

Comment:
I don't think this is used anymore, if this is the case, we should delete this line.

Reply:
Yes, the "python3" constant is still successfully used to create the virtual env to begin with, but then that virtual env contains a "Scripts/python.exe" file.